.eslintignore

```diff
@@ -0,0 +1,2 @@
+package.json
+package-lock.json
```

Comment:
nit: add a new line at the end of the file. Prettier usually handles this. 🤔

Reply:
Done ✅